blivetgui/exception_handler.py

```diff
@@ -67,14 +68,15 @@ def handle_exception(self, exc_type, exc_value, exc_traceback):
         else:
             tr_str = "".join(traceback.format_tb(exc_traceback))
 
-        if self.allow_ignore:
+        allow_report = command_exists("gnome-abrt")
```

Comment:
This exception handler is not used when running from Anaconda. Default exception handler (python-meh) should handle all exceptions in that case.